examples/classifying_shapes.ipynb

```diff
@@ -59,7 +59,8 @@
    "outputs": [],
```

Comment:
<p>maybe this is some quirk from reviewNB, but isn't the "Persistence diagrams" text redundant here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>I'm not the author of this notebook but I think this was copy-pasted from above maybe in case the reader wants to go directly to the next shape?</p>